jaxrs/WFLY-12192_optional_param.adoc

```diff
@@ -0,0 +1,64 @@
+= WFLY-12192 / EAP7-1248 Support for java.util.Optional on PathParam and QueryParam
```

Comment:
okay! In addition the current title is not accurate, because the other parameter types are also affected. So I changed title of relative JIRA issues, and created sub-task: [RESTEASY-2287 / Add complete tests for different optional paramter types](https://issues.jboss.org/browse/RESTEASY-2287).